pandas/io/pickle.py

```diff
@@ -32,12 +44,21 @@ def read_pickle(path):
     ----------
     path : string
```

Comment:
> let's leave that for another PR

It would fit well with a pull request to accomplish https://github.com/pandas-dev/pandas/issues/15008.